include/MidiClip.h

```diff
@@ -42,7 +42,7 @@ class LMMS_EXPORT MidiClip : public Clip
 public:
 	enum MidiClipTypes
 	{
-		BeatClip,
```

Comment:
I would suggest keeping BeatClip, as a distinction between a beat clip and a melody clip in a pattern (previously beat+bassline) makes sense and a BeatClip is the default for a new instrument track added to a pattern. Also, a BeatClip, as the name suggests, contains beats and no melodies.

Reply:
Good thinking, and I just realized that name conflict between `MidiClipTypes::PatternClip` and `PatternClip`. Reverted.